podvm/qcow2/misc-settings.sh

```diff
@@ -11,6 +11,31 @@
 # This ensures machine-id is generated during first boot and a unique
 # dhcp IP is assigned to the VM
 sudo echo -n > /etc/machine-id
-
 #Lock password for the ssh user (peerpod) to disallow logins
 sudo passwd -l peerpod
+
+# install required packages
+if [ "$CLOUD_PROVIDER" == "vsphere" ]
+then
+# Add vsphere specific commands to execute on remote
+    case $PODVM_DISTRO in
+	centos)
+	    #fallthrough
+	    ;&
```

Comment:
It should be installed in CentOS as well. 
https://centos.pkgs.org/8-stream/centos-appstream-x86_64/open-vm-tools-12.1.5-1.el8.x86_64.rpm.html

Reply:
@bpradipt just to be sure, you mean that we should install open-vm-tools for CentOS too right ? This is a fallthrough statement, so we will check and install for open-vm-tools too (as long as package management is configured).

Reply:
yes.. but does it require any repo enablement on the fly to install open-vm-tools?

Reply:
Other way round, if you create a CentOS image using this PR does it include open-vm-tools ?

Reply:
> yes.. but does it require any repo enablement on the fly to install open-vm-tools?

As long as you are using a CentOS stream image, you would already be setup with repos enabled for AppStream. I am not sure if there are other images you can download but if they are not Stream images, I think it's a bad idea to use the AppStream repos unless you do something like dnf upgrade && dnf install open-vm-tools.

> Other way round, if you create a CentOS image using this PR does it include open-vm-tools ?

I used this  https://cloud.centos.org/centos/8-stream/x86_64/images/CentOS-Stream-GenericCloud-8-20220913.0.x86_64.qcow2 during testing and it correctly installed open-vm-tools for vsphere.